spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
```

Comment:
Can we rename this to `comments_spec.rb`?
It being in the controllers directory already implies its a controller, right?

Reply:
it does, but I like the explicitness because that's the name of the class. Also better for fuzzy finding, since it won't get mixed up with the model spec.